README.md

```diff
@@ -175,8 +183,8 @@ $ pytest <path to dir with tests>
     "db": {
         "ENGINE": "django.db.backends.postgresql",
         "NAME": "your_db_name",
```

Comment:
Current convention requires DB_NAME as well and DB_HOST

Reply:
done, please review and resolve if ok